CHANGELOG.md

```diff
@@ -1,3 +1,198 @@
+- Fix required `cwd` field in DAP `RunTerminalArguments` type ([85411be](https://github.com/helix-editor/helix/commit/85411be), [#3240](https://github.com/helix-editor/helix/pull/3240))
+- Add LSP `workspace/applyEdit` to client capabilities ([#3012](https://github.com/helix-editor/helix/pull/3012))
+- Respect count for repeating motion ([#3057](https://github.com/helix-editor/helix/pull/3057))
+- Respect count for selecting next/previous match ([#3056](https://github.com/helix-editor/helix/pull/3056))
+- Respect count for tree-sitter motions ([#3058](https://github.com/helix-editor/helix/pull/3058))
+- Make gutters padding optional ([#2996](https://github.com/helix-editor/helix/pull/2996))
+- Support pre-filling prompts ([#2459](https://github.com/helix-editor/helix/pull/2459), [#3259](https://github.com/helix-editor/helix/pull/3259))
+- Add statusline element to display file line-endings ([#3113](https://github.com/helix-editor/helix/pull/3113))
+- Keep jump and file history when using `:split` ([#3031](https://github.com/helix-editor/helix/pull/3031), [#3160](https://github.com/helix-editor/helix/pull/3160))
+- Make tree-sitter query `; inherits <language>` feature imperative ([#2470](https://github.com/helix-editor/helix/pull/2470))
+- Indent with tabs by default ([#3095](https://github.com/helix-editor/helix/pull/3095))
+- Fix non-msvc grammar compilation on Windows ([#3190](https://github.com/helix-editor/helix/pull/3190))
+- Add spacer element to the statusline ([#3165](https://github.com/helix-editor/helix/pull/3165), [255c173](https://github.com/helix-editor/helix/commit/255c173))
+- Make gutters padding automatic ([#3163](https://github.com/helix-editor/helix/pull/3163))
+- Add `code` for LSP `Diagnostic` type ([#3096](https://github.com/helix-editor/helix/pull/3096))
+- Add position percentage to the statusline ([#3168](https://github.com/helix-editor/helix/pull/3168))
+- Add a configurable and themable statusline separator string ([#3175](https://github.com/helix-editor/helix/pull/3175))
+- Use OR of all selections when `search_selection` acts on multiple selections ([#3138](https://github.com/helix-editor/helix/pull/3138))
+- Add clipboard information to logs and the healthcheck ([#3271](https://github.com/helix-editor/helix/pull/3271))
+- Fix align selection behavior on tabs ([#3276](https://github.com/helix-editor/helix/pull/3276))
+
+Themes:
+
+- Update `tokyonight` and `tokyonight_storm` themes ([#2606](https://github.com/helix-editor/helix/pull/2606))
+- Update `solarized_light` themes ([#2626](https://github.com/helix-editor/helix/pull/2626))
+- Fix `catpuccin` `ui.popup` theme ([#2644](https://github.com/helix-editor/helix/pull/2644))
+- Update selection style of `night_owl` ([#2668](https://github.com/helix-editor/helix/pull/2668))
+- Fix spelling of `catppuccin` theme ([#2713](https://github.com/helix-editor/helix/pull/2713))
```

Comment:
Wouldn't this be a breaking change?

Reply:
Yeah good call. We should probably try to get https://github.com/helix-editor/helix/pull/3281 into this release too so we don't get a reputation for breaking catppuccin every release 😛